mlir/test/Catalyst/BufferizationTest.mlir

```diff
@@ -30,7 +30,7 @@ func.func @dbprint_val(%arg0: tensor<?xf64>) {
 
 func.func @dbprint_memref(%arg0: tensor<?xf64>) {
 
-    // CHECK: "catalyst.print"(%0) {print_descriptor} : (memref<?xf64>) -> ()
+    // CHECK: "catalyst.print"(%0) <{print_descriptor}> : (memref<?xf64>) -> ()
```

Comment:
Is this the attribute dictionary that is now printed like this?

Reply:
Honestly I don't know, I did not have to change anything expect that in the test. I could not find where the change comes from